gcloud-java-examples/src/main/java/com/google/cloud/examples/datastore/DatastoreExample.java

```diff
@@ -159,28 +198,98 @@ public void run(Transaction tx, Key userKey, String... args) {
         stBuilder.setLength(stBuilder.length() - 1);
         content = stBuilder.toString();
       }
-      IncompleteKey commentKey = IncompleteKey.builder(userKey, COMMENT_KIND).build();
-      FullEntity<IncompleteKey> comment = FullEntity.builder(commentKey)
-          .set("content", content)
-          .set("timestamp", DateTime.now())
+      return content;
+    }
+
+    @Override
+    protected String params() {
+      return "<comment>";
```

Comment:
This returns the command line help which can be more complex of a simple list of tokens to surround with `<>`. For instance we might also have info on the cardinality `?*+` or grouping `(<value1><value2>)`. This is used in other module's examples so I would keep it as is for consistency.